src/installation/guides/fde.md

```diff
@@ -176,44 +169,51 @@ and, for glibc systems only:
 
 ### Filesystem configuration
 
-The next step is editing `/etc/fstab`, which will depend on how you configured
-and named your filesystems. For this example, the file should look like this:
+Before continuing with filesystem configuration you need to get your [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier)s,
+This can be done using utilities such as: [lsblk(8)](https://man.voidlinux.org/lsblk.8) or [blkid(8)](https://man.voidlinux.org/blkid.8).
+
+The next step is editing `/etc/fstab`, 
+The UUID needed for `/etc/fstab` is the `/dev/mapper/voidvm` UUID on non-LVM configurations and `/dev/voidvm/root` UUID on LVM configurations
+
+```
+# blkid -o value -s UUID /dev/mapper/voidvm
+```
 
 ```
 # <file system>	   <dir> <type>  <options>             <dump>  <pass>
 tmpfs             /tmp  tmpfs   defaults,nosuid,nodev 0       0
-/dev/voidvm/root  /     xfs     defaults              0       0
-/dev/voidvm/home  /home xfs     defaults              0       0
-/dev/voidvm/swap  swap  swap    defaults              0       0
+UUID=ADD-UUID-FROM-BLKID-COMMAND-HERE   /     btrfs   defaults              0       0
```

Comment:
Drop these changes.

Reply:
Why? What is the point of dropping a change that tells users how to get their UUIDs?

Also the /etc/fstab should use the UUIDs not /dev/voidvm/root.